src/apps/applets/tests/test_applet_activity_items.py

```diff
@@ -263,6 +268,25 @@ async def test_create_applet__activity_with_conditional_logic(
             by_alias=True
         )
 
+    async def test_create_applet__activity_with_paragraph_and_phrase_builder(
```

Comment:
This test is a copy of test above test_create_applet__activity_with_conditional_logic

Reply:
what this test should be testing? it is a duplicate of a existing tests for conditional logic not related to phrasal template